sdk/monitor/azure-monitor-ingestion/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 ### Bugs Fixed
 
 ### Other Changes
+  - Python 3.6 is no longer supported. Please use Python version 3.7 or later. For more details, see [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy).
```

Comment:
No need for this.

This is your first GA release. In other words, we "never" supported py 3.6. :)

Reply:
Thanks, I removed that line.